tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -95,6 +95,8 @@ Define {
 	DMIC0_PCM_0_NAME		"DMIC SFX1"
 	DMIC0_PCM_1_NAME		"DMIC SFX2"
 	SRC_DOMAIN			"default"
+	# Keep DP_SRC_CORE_ID == SSP2_PCM_CORE_ID, no nested define resolvation ATM
```

Comment:
ok, so is this a test topology that reproduces the issue ?

Reply:
Yes. The same commit exists in the DP multi core test PR: https://github.com/thesofproject/sof/pull/10009